transaction/account/client.go

```diff
@@ -115,17 +135,17 @@ func (c *client) sendRequestAwait(ctx *Context, body body) (*cc.TransactionOutco
 	}
 	nonce, err := c.cli.GetNextAccountNonce(ct, ctx.Sender)
 	if err != nil {
-		return nil, fmt.Errorf("unable to get next nonce: %w", err)
+		return "", fmt.Errorf("unable to get next nonce: %w", err)
 	}
 	// TODO what to do in case of false
```

Comment:
There is nothing wrong in this case, but it might mean your transaction will fail. So depending on how critical your transaction is, it might want to not send it.